Vagrantfile

```diff
@@ -52,35 +62,39 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # backing providers for Vagrant. These expose provider-specific options.
   # Example for VirtualBox:
   #
-  # config.vm.provider :virtualbox do |vb|
-  #   # Don't boot with headless mode
+  # config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
   #   vb.gui = true
   #
-  #   # Use VBoxManage to customize the VM. For example to change memory:
-  #   vb.customize ["modifyvm", :id, "--memory", "1024"]
+  #   # Customize the amount of memory on the VM:
+  #   vb.memory = "1024"
   # end
   #
-  # View the documentation for the provider you're using for more
+  # View the documentation for the provider you are using for more
   # information on available options.
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "4096"
+    vb.cpus = "4"
```

Comment:
4gb of RAM and 4 cores seems like a bit much for development
Can you confirm that this will not have any problems on a machine that physically can not meet the requirements?

[Stackstorm docs](https://docs.stackstorm.com/install/system_requirements.html) mention 2 cores and 1gb of ram is fine for dev